pkg/api/schema/sql_exec_result.go

```diff
@@ -0,0 +1,17 @@
+package schema
+
+// LastInsertedPk returns a map in which the keys are the names of the tables and the values are the primary keys of the last inserted rows.
+func (er *SQLExecResult) LastInsertedPk() map[string]*SQLValue {
```

Comment:
I don't find this two methods necessary. I'd just use the attribute execResult.LastInsertedPKs
if execResult is nil it means there was an error first